extensions/binding/org.eclipse.smarthome.binding.ntp.test/src/test/java/org/eclipse/smarthome/binding/ntp/test/NtpOSGiTest.java

```diff
@@ -217,7 +217,8 @@ public void testDateTimeChannelTimeZoneUpdate() {
         String testItemState = getItemState(ACCEPTED_ITEM_TYPE_DATE_TIME).toString();
         assertFormat(testItemState, DateTimeType.DATE_PATTERN_WITH_TZ_AND_MS);
         ZonedDateTime timeZoneFromItemRegistry = ((DateTimeType) getItemState(ACCEPTED_ITEM_TYPE_DATE_TIME))
-                .getZonedDateTime();
+                .getZonedDateTimeAtZone(ZoneId.of(TEST_TIME_ZONE_ID));
```

Comment:
To be honest, I don't really get this change. The purpose of this test is to verify the inverse of "The dateTime channel was not updated with the right timezone", i.e. that the channel _was_ updated with the right time zone. Now you have to manually convert the time zone to the desired one. Isn't that obfuscating some other flaw?